### PR TITLE
Multiple Reactions icon align with text

### PR DESCRIPTION
**Pull request by rt4914**
## What type of PR is this? (check all applicable)

- [x] Refactor

## Description

Element alignment on #index. Example: https://forem.team/t/allhands (both regular and dark mode). The alignment looks off because of off the icons are aligned with the text. 

the multi-reaction count is not a clickable button as a comment for the article card.

Feb-08-2023 09-39-11

## Related Tickets & Documents

- Related Issue #19064

## QA Instructions, Screenshots, Recordings

| Before  | After |
| ------------- | ------------- |
| Screenshot 2023-02-07 at 10 30 42 PM | Screenshot 2023-02-07 at 10 30 13 PM |
| Feb-08-2023 09-39-11 | Screenshot 2023-02-08 at 10 52 18 AM |

On click on the `reactions + text` button in above image, it opens the article.

https://user-images.githubusercontent.com/9396084/217442343-06eb97de-d65a-44d9-b34d-3bdd0029facf.mov

### Added border to reactions icon 
Screenshot 2023-02-08 at 12 07 52 PM (1) Screenshot 2023-02-08 at 12 19 26 PM

### UI accessibility concerns?

No

## Added/updated tests?

- [x] No

## [Forem core team only] How will this change be communicated?

- [x] This change does not need to be communicated

### CODE REVIEW

**Comment by anujbhavsar96** [comment]
Overall, alignment stuff and clickable element looks perfect🚀

---

@rt4914 adding these two small tweaks here as we discussed

**1. Give multi-reaction icons border**
Screenshot 2023-02-08 at 11 21 13 AM


**2. Change the icon order - Left most should be on top of the stack**
Screenshot 2023-02-08 at 11 26 13 AM

**Comment by rt4914** [comment]
> Overall, alignment stuff and clickable element looks perfect🚀
> 
> @rt4914 adding these two small tweaks here as we discussed
> 
> **1. Give multi-reaction icons border** Screenshot 2023-02-08 at 11 21 13 AM
> 
> **2. Change the icon order - Left most should be on top of the stack** Screenshot 2023-02-08 at 11 26 13 AM

@anujbhavsar96 Update:

1. Added border and updated screenshots
2. Added as task to #19064

**Review by anujbhavsar96 — APPROVED** [review]
All three changes look good to me 🚀

File: app/assets/stylesheets/views/article.scss [hunk 1]
```diff
@@ -460,22 +460,26 @@
...
 
 .multiple_reactions_aggregate {
-  padding-right: 1em;
+  display: flex;
+  align-items: center;
 
   .crayons_icon_container {
     display: inline-block;
     background-color: var(--body-bg);
+    border: solid;
```
Comment (Ridhwana) [inline app/assets/stylesheets/views/article.scss:469]: (suggestion): You could use the [border css shorthand property](https://developer.mozilla.org/en-US/docs/Web/CSS/border) here:

```
border: 2px solid var(--card-bg)
```

and remove `border: solid;`, ` border-color: var(--card-bg);`, `border-width: 2px;`

File: app/views/articles/_single_story.html.erb [hunk 2]
```diff
@@ -121,18 +121,20 @@
...
-                  <span class="crayons_icon_container">
-                    <%= crayons_icon_tag(reaction_type.icon, width: "18px", height: "18px", native: true, aria_hidden: true) %>
-                  </span>
-                <% end %>
-                <span class="aggregate_reactions_counter"><%= t("views.reactions.summary.count_html",
-                                                                count: story.public_reactions_count,
-                                                                start: tag("span", { class: %w[hidden s:inline] }, true),
-                                                                end: "</span>".html_safe) %></span>
-              </div>
+              <a href="<%= story.path %>" class="crayons-btn crayons-btn--s crayons-btn--ghost crayons-btn--icon-left" data-reaction-count data-reactable-id="<%= story.id %>" aria-label="<%= t("views.articles.comments.aria_label", title: story.title) %>">
```
Comment (Ridhwana) [inline app/views/articles/_single_story.html.erb:124]: Nice catch on adding the link here.

**Review by Ridhwana — APPROVED** [review]
Code looks good ✨Left one suggestion for the CSS.

File: app/assets/stylesheets/views/article.scss (hunk 1 above)
Comment (rt4914) [reply app/assets/stylesheets/views/article.scss]: Done. Thanks.

File: app/views/articles/_single_story.html.erb (hunk 2 above)
Comment (rt4914) [reply app/views/articles/_single_story.html.erb:124]: Thanks.

**Pull request merged**